server/src/utils/index.js

```diff
@@ -34,8 +39,26 @@ const verifySignature = (messageString, signatureHex, bitmarkAccountNumber) => {
   }
 };
 
+const validJWT = (token) => {
+  return new Promise((resolve) => {
+    jwt.verify(token, certJWT, {
+      algorithms: ['RS256'],
+    }, (error, decoded) => {
+      if (error) {
+        return resolve();
+      }
+      if (decoded && (decoded.iat * 1000) < Date.now() && (decoded.exp * 1000) > Date.now()) {
```

Comment:
Should add a comment here to explain what is business logic here

Reply:
done